docs/dictionaries/pinyinWordlistReadme.txt

```diff
@@ -0,0 +1,11 @@
+Chinese Pinyin word list by Vkiki2233:
```

Comment:
Remove all unprintable characters in a separate commit.

Reply:
It is very difficult to find the Pinyin of the split phrases. Giving up the latter.